.github/workflows/java-ci.yml

```diff
@@ -108,7 +108,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        jvm: [11, 17, 21]
+        jvm: [17, 21]
```

Comment:
This is for `build-javadoc`. Not able to do `build-javadoc` using java 11 because Spark 4.0 is build using java 17.

Reply:
Is there a reason to do build-javadoc on more than one JVM anyway?